tests/components/motioneye/test_camera.py

```diff
@@ -4,7 +4,7 @@
 from typing import Any
 from unittest.mock import AsyncMock, Mock
 
-from aiohttp import web  # type: ignore
+from aiohttp import web
```

Comment:
Tests are actually not type checked as part of CI, but why not.

Reply:
Noticed that -- but tests were _mostly_ type compliant, so might as well get it across the finish line!